Library/Formula/duck.rb

```diff
@@ -0,0 +1,29 @@
+class Duck < Formula
+  homepage "https://duck.sh/"
+  url "https://dist.duck.sh/duck-src-16336.tar.gz"
+  version "4.6.2.16336"
+  head "https://svn.cyberduck.io/trunk/"
+  sha1 "377f8ca3d111416a20122e0e5f41f189c5eda99e"
+
+  depends_on :java => ["1.7", :build]
+  depends_on :xcode => :build
+  depends_on "ant" => :build
+  depends_on "openssl"
+
+  def install
+    system "ant", "-Dbuild.compile.target=1.7", "-Drevision=16336", "cli"
+    system "install_name_tool", "-change", "/usr/lib/libcrypto.0.9.8.dylib", "/usr/local/opt/openssl/lib/libcrypto.dylib", "build/duck.bundle/Contents/Frameworks/libPrime.dylib"
+    system "install_name_tool", "-change", "/usr/lib/libcrypto.0.9.8.dylib", "/usr/local/opt/openssl/lib/libcrypto.dylib", "build/duck.bundle/Contents/Frameworks/librococoa.dylib"
+    libexec.install Dir["build/duck.bundle/*"]
+    bin.install_symlink "#{libexec}/Contents/MacOS/duck" => "duck"
+  end
+
+  test do
+    unless "Cyberduck 4.6.2 (16336)\n".eql? %x(#{bin}/duck -version)
+      fail "Version mismatch"
+    end
```

Comment:
Seems that should be a unit test in the build process itself. We just want to check that e.g. it can do what it needs to. If the version information is incorrect I'm not sure we'd mind, honestly.